README.md

```diff
@@ -1,56 +1,31 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
+<img src="https://wakatime.com/badge/user/be79098d-c59e-407c-8952-0f274bb9e265/project/508ec814-cdd5-44c1-9678-51cda72acb32.svg" />
 
-# Project Name
+# Leaderboard
```

Comment:
- [ ] In your README kindly add instructions on how to set up and run your app locally ie from cloning to `npm install` to `npm start`